include/nccl_ofi_param.h

```diff
@@ -179,9 +179,10 @@ OFI_NCCL_PARAM_INT(disable_native_rdma_check, "DISABLE_NATIVE_RDMA_CHECK", 0);
 OFI_NCCL_PARAM_INT(disable_gdr_required_check, "DISABLE_GDR_REQUIRED_CHECK", 0);
 
 /*
- * Maximum size of a message in bytes before message is multiplexed
- */
-OFI_NCCL_PARAM_INT(round_robin_threshold, "ROUND_ROBIN_THRESHOLD", (256 * 1024));
+ * Minimum data size in bytes before the message is to be striped across multiple 
+ * rails.
+*/
+OFI_NCCL_PARAM_INT(min_stripe_size, "MIN_STRIPE_SIZE", (210 * 1024));
```

Comment:
We should leave this threshold at 256KiB and come back to tuning it after the scheduler changes land. Better to track changes in perf and also avoid imbalance in layers below by sticking to power of two... for when a single stripe gets further segmented across hardware resources.

Something to address outside this PR: I realized we've been using _INTs for all numerical params, sizes should be size_t/unsigned longs to line up with the rest of the interfaces.

Reply:
Done. Changed the threshold to 256KB. I will create a new PR to change the _INTs to size_t